File: Python/Code/1000/1065 - 한수.py
n = int(input())
result = 0
for i in range(1, n+1):
	if i < 100:
		result += 1
	else:
		a = i // 100
		c = i % 10
		b = ((i - c) // 10) % 10
		if a - b == b - c:
			result += 1
print(result)

File: Java/Code/1000/1065 - 한수.java
import java.util.Scanner;

public class Main {
	public static void main(String[] args) {
		Scanner sc = new Scanner(System.in);
		int n = sc.nextInt();
		int result = 0;
		for (int i = 1; i <= n; i++)
			if (i < 100)
				result++;
			else if ((i / 100 - i / 10 % 10) == (i / 10 % 10 - i % 10))
				result++;
		System.out.println(result);
		return;
	}
}